grid_map_visualization/src/GridMapVisualization.cpp

```diff
@@ -6,165 +6,170 @@
  *	 Institute: ETH Zurich, ANYbotics
  */
 
-#include "grid_map_visualization/GridMapVisualization.hpp"
 #include <grid_map_core/GridMap.hpp>
 #include <grid_map_ros/GridMapRosConverter.hpp>
 
-using namespace std;
-using namespace ros;
+#include <string>
+#include <vector>
+#include <memory>
+#include <chrono>
```

Comment:
Alphabetical order, can you check the other includes as well?

Reply:
Done.